src/.vuepress/config/sideBar.js

```diff
@@ -83,6 +83,7 @@ module.exports = {
                 '/guides/search',
                 '/guides/webreader',
                 '/guides/tachiyomi',
+				'/guides/paperback',
```

Comment:
```suggestion
                '/guides/paperback',
```

Spaces, not tabs !

Reply:
Woups, sorry, I didn't check and missed the issue 😞